app/frontend/src/signOutAfterInactivity.js

```diff
@@ -0,0 +1,28 @@
+var timeoutInMinutes = 30
+
+var timeoutId; 
+
+function resetTimer() { 
+    window.clearTimeout(timeoutId)
+    startTimer();
+}
+
+function clickSignOut() {
+    document.getElementById('sign_out').click();
+}
+
+function startTimer() { 
+    var timeoutInMilliseconds = timeoutInMinutes * 60000;
```

Comment:
whilst testing is limited you can do with this, pass `timeoutInMinutes` into this function as an argument and will make testing this possible, other wise the test will take 30mins to run and we would test on a much shorter timeout. i suggest we pair on a few tests here once algolia is merged (and test suite and linting is available). but for now id be happy that this isnt tested